mypy/test/data.py

```diff
@@ -191,7 +188,7 @@ def parse_test_cases(
     return out
 
 
-class DataDrivenTestCase(TestCase):
+class DataDrivenTestCase(ProtoTestCase):
```

Comment:
This is kind of confusing, since there seems to be no longer anything in this class about how an individual test case is run. This at least needs a docstring.